src/__snapshots__/compile.test.ts.snap

```diff
@@ -9,9 +9,9 @@ console.log(A)
 
 exports[`build fails when a non-existent relative import is specified 1`] = `"Could not resolve './__doesNotExist.js' from src/__test__/compile/relativeImportNotFound.js"`;
 
-exports[`build fails when an absolute import is specified 1`] = `/foo.js is imported by <cwd>/src/__test__/compile/absoluteImport.js, but absolute imports are disallowed`;
+exports[`build fails when an absolute import is specified 1`] = `"/foo.js is imported by <cwd>/src/__test__/compile/absoluteImport.js, but absolute imports are disallowed"`;
```

Comment:
Quotes are added because the previous test serializer didn't call the `serialize` of `print(val, serialize)`; but now it is required to have working the `jest-serializer-path`